### PR TITLE
FEAT: rebranding for migration notebooks

### PR DESCRIPTION
**Pull request by andrewferlitsch**

### CODE REVIEW

File: ai-platform-unified/notebooks/unofficial/migration/UJ1 unified AutoML Vision Image Classification.ipynb [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "id": "title:migration,new"
    },
    "source": [
-    "# AI Platform (Unified) SDK: AutoML image classification model\n",
+    "# Vertex SDK: AutoML image classification model\n",
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/migration/UJ1 unified AutoML Vision Image Classification.ipynb:32]: According to @sasha-gitg  `Vertex SDK` should be `Vertex SDK for Python`

File: ai-platform-unified/notebooks/unofficial/migration/UJ10 unified Custom Training Prebuilt Container SKLearn.ipynb [hunk 2]
```diff
@@ -187,13 +187,13 @@
...
     "You can also change the `REGION` variable, which is used for operations\n",
-    "throughout the rest of this notebook.  Below are regions supported for AI Platform (Unified). We recommend when possible, to choose the region closest to you.\n",
+    "throughout the rest of this notebook.  Below are regions supported for Vertex AI. We recommend when possible, to choose the region closest to you.\n",
     "\n",
     "- Americas: `us-central1`\n",
     "- Europe: `europe-west4`\n",
     "- Asia Pacific: `asia-east1`\n",
     "\n",
-    "You cannot use a Multi-Regional Storage bucket for training with AI Platform. Not all regions provide support for all AI Platform services. For the latest support per region, see [Region support for AI Platform (Unified) services](https://cloud.google.com/ai-platform-unified/docs/general/locations)\n",
+    "You cannot use a Multi-Regional Storage bucket for training with Vertex AI. Not all regions provide support for all Vertex AI services. For the latest support per region, see [Region support for Vertex AI services](https://cloud.google.com/ai-platform-unified/docs/general/locations)\n",
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/migration/UJ10 unified Custom Training Prebuilt Container SKLearn.ipynb:196]: Change `https://cloud.google.com/ai-platform-unified/docs/general/locations` to `https://cloud.google.com/vertex-ai/docs/general/locations`

Also do for other affected links.

**Comment by ivanmkc** [comment]
To replace the links, you can try this:

`grep -rl 'ai-platform-unified' ./ai-platform-unified/notebooks/unofficial/migration/* | gxargs -d '\n' gsed -i 's/com\/ai-platform-unified/com\/vertex-ai/g'`

You'll have to install `gsed` though

**Comment by andrewferlitsch** [comment]
branch problems

**Pull request closed without merging**